src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.shared.cs

```diff
@@ -0,0 +1,37 @@
+﻿#if !(IOS || ANDROID || MACCATALYST || WINDOWS)
```

Comment:
Why not put this into a `.netstandard.cs` file?

Reply:
it is not ready yet. I can do it after Popup is merged